resources/snakefiles/qc.smk

```diff
@@ -0,0 +1,26 @@
+# fastqc each unit
+rule fastqc:
+    input:
+        lambda wildcards: get_read(wildcards.sample,
```

Comment:
If I'm reading this correctly, the same sample on two different runs (i.e. "amy lane_1" and "amy lane_2", above) are being treated essentially as two different samples, right? So why not just give them different Sample_IDs?

Reply:
Just for this example step -- potentially useful for diagnosing technical issues. (Say you ran the same library on two flow cells, with the second time to 'top up' the reads from the first time. It's a good idea to have more granular QC information for those raw FASTQs to be able to diagnose issues.)

Reply:
I see. I understand that you'd want to keep them separate for diagnostic reasons, but you're saying that there may also be instances where you'd want to treat both of amy's samples as one sample (like co-assembling them or something).